### DISCUSSION TITLE
NvimTree opening document on the wrong side

### DISCUSSION THREAD

**Discussion by lockieluke** [nvim-tree/nvim-tree.lua#1496] [General]
https://user-images.githubusercontent.com/25424409/183872555-f5b3ae01-252c-46cd-af84-f9d9aae00ade.mov

nvim tree is vspliting the first document on the wrong side, it would be working correctly again after opening the first document

**Comment by alex-courtis** [comment]
Edit: just saw #1497. Please see comments there.
